pages/company/brand-guidelines.mdx

```diff
@@ -6,9 +6,9 @@ For the visionary in search of IoT solutions, Digital Fortress empowers your jou
 ### Brand Audiences
 
 - **Current Customers**: International partners in Australia, the United States, and Europe who have existing relationships with Digital Fortress and actively collaborate with the company.
-- **Target Customers**: Desired partners located in Singapore and Dubai, where Digital Fortress aims to establish new partnerships and expand its presence.
+- **Target Customers**: The SMEs, international partners that use English as the common language for communication especialy in United State, Singapore. They have already established their business models, production processes and are seeking to scale their management, human resources system, product offerings through advanced IoT applications.

```

Comment:
>that use English as the common language for communication especially
I just want to clarify the market where we are targeting, we shouldn't mention this in the actual content, An

Reply:
got it anh